allauth/socialaccount/providers/unidays/views.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+from allauth.socialaccount.providers.oauth2.views import (
+    OAuth2Adapter,
+    OAuth2CallbackView,
+    OAuth2LoginView,
+)
+
+from .provider import UNiDAYSProvider
+
+
+class UNiDAYSOAuth2Adapter(OAuth2Adapter):
+    provider_id = UNiDAYSProvider.id
+    access_token_url = "https://account.myunidays.com/oauth/token"
+    authorize_url = "https://account.myunidays.com/oauth/authorize"
+    profile_url = "https://account.myunidays.com/oauth/userinfo "
+
+    def complete_login(self, request, app, token, **kwargs):
+        resp = requests.get(
+            self.profile_url,
+            headers={"Authorization": f"Bearer: {token.token}"},
```

Comment:
This (f-string) will break the test suite as it still includes py3.5

Reply:
Have removed the f-string :)